components/progressbar/metadata/meter.yml

```diff
@@ -18,7 +18,7 @@ examples:
     markup: |
       <div class="spectrum-Examples" style="flex-direction: column;">
         <div class="spectrum-Examples-item">
-          <div class="spectrum-ProgressBar spectrum-Meter--sizeS is-positive" value="50" role="progressbar" aria-valuenow="50" aria-valuemin="0" aria-valuemax="100" style="width: 200px;">
+          <div class="spectrum-ProgressBar spectrum-Meter spectrum-Meter--sizeS is-positive" value="50" role="progressbar" aria-valuenow="50" aria-valuemin="0" aria-valuemax="100">
```

Comment:
So it looks like you've added the need for an additional class here so we should probably call that out in the Migration Guide